<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmEmailTool</class>
 <widget class="QWidget" name="frmEmailTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labSenderAddr">
     <property name="text">
      <string>用户名:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtSenderAddr">
     <property name="text">
      <string>feiyangqingyun@126.com</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labSenderPwd">
     <property name="text">
      <string>密码:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="txtSenderPwd">
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="labServer">
     <property name="text">
      <string>服务器:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QComboBox" name="cboxServer">
     <item>
      <property name="text">
       <string>smtp.163.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>smtp.126.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>smtp.qq.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>smt.sina.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>smtp.sohu.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>smtp.139.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>smtp.189.com</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="labPort">
     <property name="text">
      <string>端口:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QComboBox" name="cboxPort">
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>465</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>587</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QCheckBox" name="ckSSL">
     <property name="text">
      <string>SSL</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QPushButton" name="btnSend">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labTitle">
     <property name="text">
      <string>标  题:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labFileName">
     <property name="text">
      <string>附  件:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labContent">
     <property name="text">
      <string>正  文:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="9">
    <widget class="QPushButton" name="btnSelect">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="8">
    <widget class="QLineEdit" name="txtFileName"/>
   </item>
   <item row="5" column="1" colspan="9">
    <widget class="QTextBrowser" name="txtContent">
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;、最短的爱情哲理小说：“&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;你应该嫁给我啦？&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;” “ &lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;不&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;” &lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;于是他俩又继续幸福地生活在一起&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;！&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;、近年来中国最精彩的写实小说，全文八个字：&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:16pt; font-weight:600; font-style:italic; color:#ff007f;&quot;&gt;此地钱多人傻速来&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;  据说是发自杭州市宝石山下一出租房的汇款单上的简短附言，是该按摩女给家乡妹妹汇 款时随手涂鸦的，令无数专业作家汗颜！&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;3&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;、最短的幽默小说 《夜》 男：疼么？女：恩！男：算了？女：别！&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;4&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;、最短的荒诞小说：有一个面包走在街上，它觉得自己很饿，就把自己吃了。&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;5&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;、世界最短言情小说：他死的那天，孩子出生了。&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;6&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;、世界最短武侠小说：&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt; font-weight:600; color:#00aa00;&quot;&gt;高手被豆腐砸死了&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;。&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;7&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;、世界最短科幻小说：最后一个地球人坐在家里，突然响起了敲门声。&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;8&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;、世界最短悬疑小说：生，死，生。&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;9&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;、世界最短推理小说：他死了，一定曾经活过。 &lt;br /&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;0&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt;&quot;&gt;、世界最短恐怖小说：惊醒，身边躺着自己的尸体。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="txtTitle">
     <property name="text">
      <string>测试邮件</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="labReceiverAddr">
     <property name="text">
      <string>收件人:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5" colspan="5">
    <widget class="QLineEdit" name="txtReceiverAddr">
     <property name="text">
      <string>feiyangqingyun@163.com;517216493@qq.com</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
